src/test/java/io/javalin/examples/SSEExample.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.examples;
+
+import io.javalin.Javalin;
+import io.javalin.serversentevent.EventSource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class SSEExample {
+
+    public static void main(String[] args) throws InterruptedException {
+        List<EventSource> eventSources = new ArrayList<EventSource>();
+
+        Javalin app = Javalin.create().start( 7000 );
+        app.get( "/", ctx ->
+                ctx.html(
+                        ""
+                                + "<script>" +
+                                "var sse = new EventSource(\"http://localhost:7000/sse/1\");" +
+                                "sse.addEventListener(\"hi\", data => console.log(data));"
+                                + "</script>"
+                )
+        );
+
+        app.sse( "/sse/:id", sse -> {
+            sse.sendEvent( "connect", "Connected!" );
+            eventSources.add( sse );
+            sse.onClose( eventSource -> {
+                eventSources.remove( eventSource );
+                return null;
```

Comment:
Why `return null`?

Reply:
If I remember correctly, that's an interop problem. You can use

```java
@FunctionalInterface
public interface SseClose {
    void handle(@NotNull EventSource eventSource) throws Exception;
}
```

instead of the typealias, and Java will see it correctly. You can then write `sse.onClose(eventSources::remove);` instead. 

But if you do this, Kotlin won't recognize the method, so you also need to convert the `EventSource` class to Java. This is the reason most of the public facing API of Javalin is written in Java.